### PR TITLE
Labs10 frontend surveyspage

### PR DESCRIPTION
**Pull request by BrandCam**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by latifahpresident — APPROVED** [review]
Looks good

File: frontend/src/pages/Surveys/SurveyCard.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            <div>  
+                <div className ='survey-left-content'>
+                    <h3>{`${name}`}</h3>
+                    <Link to='#'>
+                        <Button text="View Survey Responces" color='var(--color-accent)'></Button>
+                    </Link>
+                </div>
+                <div className ='survey-rigth-content'>
+                    <p>Responces</p>
+                    {/* need data */}
```
Comment (ghost) [inline frontend/src/pages/Surveys/SurveyCard.tsx:23]: There are some typos here - you probably want it to say 'survey-right-content' and 'Responses'.

File: frontend/src/pages/Surveys/types.ts [hunk 2]
```diff
@@ -4,4 +4,7 @@ export interface Survey {
     survey: any,
     id: number,
     name: string,
-}
\ No newline at end of file
+    isGuest: number,
+}
```
Comment (ghost) [inline frontend/src/pages/Surveys/types.ts:8]: Is this supposed to be a number? I thought maybe it would rather be boolean.

**Review by ghost — APPROVED** [review]
A couple of minor things - check my comments. Great work setting this up!

File: frontend/src/pages/Surveys/types.ts (hunk 2 above)
Comment (BrandCam) [reply frontend/src/pages/Surveys/types.ts:8]: the endpoint returns a number, 1 or 0. no idea why it isnt true or false

File: frontend/src/pages/Surveys/SurveyCard.tsx (hunk 1 above)
Comment (BrandCam) [reply frontend/src/pages/Surveys/SurveyCard.tsx:23]: yea ill have to fix those

**Pull request merged**